src/SMAIL-GPL.xml

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SMAIL-GPL"
+   name="SMAIL GENERAL PUBLIC LICENSE" listVersionAdded="3.26">

```

Comment:
I would lowercase "General Public License" for the name, and `listVersionAdded` should be 3.26.0.